lib/components/logging.js

```diff
@@ -191,24 +191,24 @@ if (winston) {
 else {
     // We don't have winston, so just log to the console.
     instance = {
-        get: (name) => {
+        Get: (name) => {
             const logWrapper = new LogWrapper();
             // Console has all the functions already.
             logWrapper.setLogger(console);
             return logWrapper;
         },
-        configure: () => {
+        Configure: () => {
             // No-op this.
         },
     };
 }
 
 module.exports = {
     get: (name) => {
-        return instance.get(name);
+        return instance.Get(name);
     },
     configure: (config) => {
-        instance.configure(config);
+        instance.Configure(config);
```

Comment:
these both should be lowercase though :( Why was the build complaining about it?